llvm/test/Transforms/LoopVectorize/AArch64/interleave_count.ll

```diff
@@ -103,5 +341,230 @@ for.end:
   ret void
 }
 
+; TODO: For a loop with a profile-guided estimated TC of 48, when the auto-vectorizer chooses VF 16, 
+; it should conservatively choose IC 1 so that the vector loop runs twice at least
+; CHECK: remark: <unknown>:0:0: vectorized loop (vectorization width: 16, interleaved count: 3)
+define void @loop_with_profile_tc_48(ptr noalias %p, ptr noalias %q, i64 %n) {
```

Comment:
Might be good to split off the tests with profile info to a separate file,  as the number of tests now has grown substantially.